public/styles.css

```diff
@@ -23,3 +31,14 @@ body {
   font-family: futura;
   font-size: 3em;
 }
+
+input#toggle {
+  position: absolute;
+  top: -9999px;
+  left: -9999px;
+}
+
+input#toggle:checked + .wrapper {
+  background: url("catcloud.gif");
+  background-color: #000;
```

Comment:
@lpassamano how do you feel about adding
```
  background-size: cover;
  background-repeat: no-repeat;
```
here?

Reply:
@kliph I think we talked about this irl, but I agree with the update!